deployment/src/main/java/com/datastax/oss/quarkus/deployment/internal/CassandraClientProcessor.java

```diff
@@ -100,7 +102,9 @@
         // timestamp generators
         new ReflectiveClassBuildItem(true, true, AtomicTimestampGenerator.class.getName()),
         new ReflectiveClassBuildItem(true, true, ThreadLocalTimestampGenerator.class.getName()),
-        new ReflectiveClassBuildItem(true, true, Publisher.class.getName()));
+        new ReflectiveClassBuildItem(true, true, Publisher.class.getName()),
+        // authentication
+        new ReflectiveClassBuildItem(true, true, PlainTextAuthProvider.class.getName()));
```

Comment:
Good catch! It's a pitty that we cannot (yet) use TestContainers with authentication. Let's add all the default auth providers that ship with the driver.

```suggestion
        // reactive streams
        new ReflectiveClassBuildItem(true, true, Publisher.class.getName()),
        // authentication
        new ReflectiveClassBuildItem(true, true, PlainTextAuthProvider.class.getName()));
        new ReflectiveClassBuildItem(true, true, DseGssApiAuthProvider.class.getName()));
        new ReflectiveClassBuildItem(true, true, ProgrammaticPlainTextAuthProvider.class.getName()));
        new ReflectiveClassBuildItem(true, true, ProgrammaticDseGssApiAuthProvider.class.getName()));
```